code/game/objects/items/devices/PDA/PDA_types.dm

```diff
@@ -17,6 +17,26 @@
 		if(istype(cart) && cart.charges < 5)
 			cart.charges++
 
+//Mime PDA sends "silent" messages.
+/obj/item/pda/mime
+	name = "mime PDA"
+	default_cartridge = /obj/item/cartridge/virus/mime
+	inserted_item = /obj/item/toy/crayon/mime
+	icon_state = "pda-mime"
+	desc = "A portable microcomputer by Thinktronic Systems, LTD. The hardware has been hard coded to take the vow of silence."
```

Comment:
the hardware itself doesn't take a vow, maybe "The hardware has been modified for compliance with vows of silence."

Reply:
i'll change it, you got a good point